support.html

```diff
@@ -76,19 +76,29 @@ <h2>Chat</h2>
             For quick questions you can reach us
             in the <a href="https://gitter.im/openwisp/general">OpenWISP General</a> chat.
         </p>
+            Developers and contributors coordinate with one another using
+            <a href="https://gitter.im/openwisp/development">OpenWISP Development</a> chat.
         <p>
             Since we're on different timezones, it may happen that nobody
             can answer your question while you're active in the chat.<br>
             If that happens, try to send the same to question to the
             <a href="https://groups.google.com/d/forum/openwisp" target="_blank">mailing list</a>,
             which allows a more asynchronous communication.
         </p>
+        <div class="Chat_buttons">
         <p class="center buttons">
           <a href="https://gitter.im/openwisp/general"
              class="ui big inverted button red" target="_blank">
-            <i class="comment icon"></i> Join the OpenWISP Chat
+            <i class="comment icon"></i> Join the support chat
           </a>
         </p>
+         <p class="center buttons">
+          <a href="https://gitter.im/openwisp/development"
+             class="ui big inverted button red" target="_blank">
+            <i class="comment icon"></i> Join the dev chat
+          </a>
+        </p>
```

Comment:
please simply add the new `<a>` in the existing `<p>` element and nothing more (no new div, no flex).